js/src/edit-block/components/settings/textarea-array.js

```diff
@@ -40,6 +46,28 @@ const TextareaArray = ( { handleOnChange, setting, value } ) => {
 				} }
 				value={ textAreaValue }
 			/>
+			{ Boolean( setting.help )
+				? (
+					<p className={ helpClass }>
+						{ setting.help }
+					</p>
+				) : (
+					<>
+						<p className={ helpClass }>
+							{ __(
+								'Each choice on a new line. To specify the value and label separately, use this format:',
+								'genesis-custom-blocks'
+							) }
+						</p>
+						<p className={ helpClass }>
```

Comment:
Before, this was in `setting.help`, but it has markup.

So instead of doing `__dangerouslySetInnerHTML={ setting.default }`, this has the markup in the React file.